Add minecraft-exporter to list of exporter

dirien opened the pull request:
Signed-off-by: Engin Diri <engin.diri@mail.schwarz>

<!--
    Please sign CNCF's Developer Certificate of Origin and sign-off your commits by adding the -s / --sign-off flag to `git commit`

    More information on both of those can be found at https://github.com/apps/dco

    If you are proposing a new integration, exporter, or client library, please include representative sample output.
 -->
Please add this https://github.com/dirien/minecraft-prometheus-exporter minecraft exporter to your list of exporter.

Sample output

```bash
# HELP minecraft_prometheus_exporter_animals_bred_total The number of times the player bred two mobs.
# TYPE minecraft_prometheus_exporter_animals_bred_total counter
minecraft_prometheus_exporter_animals_bred_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_aviate_one_cm_total The total distance traveled by elytra.
# TYPE minecraft_prometheus_exporter_aviate_one_cm_total counter
minecraft_prometheus_exporter_aviate_one_cm_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_bell_ring_total The number of times the player has rung a Bell.
# TYPE minecraft_prometheus_exporter_bell_ring_total counter
minecraft_prometheus_exporter_bell_ring_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_blocks_mined_total Statistic related to the number of blocks a player mined
# TYPE minecraft_prometheus_exporter_blocks_mined_total counter
minecraft_prometheus_exporter_blocks_mined_total{block="clay",player="ediri"} 1
minecraft_prometheus_exporter_blocks_mined_total{block="coal_ore",player="ediri"} 13
minecraft_prometheus_exporter_blocks_mined_total{block="crafting_table",player="ediri"} 1
minecraft_prometheus_exporter_blocks_mined_total{block="dirt",player="ediri"} 19
minecraft_prometheus_exporter_blocks_mined_total{block="grass",player="ediri"} 31
minecraft_prometheus_exporter_blocks_mined_total{block="grass_block",player="ediri"} 17
minecraft_prometheus_exporter_blocks_mined_total{block="oak_leaves",player="ediri"} 1
minecraft_prometheus_exporter_blocks_mined_total{block="oak_log",player="ediri"} 5
minecraft_prometheus_exporter_blocks_mined_total{block="oxeye_daisy",player="ediri"} 1
minecraft_prometheus_exporter_blocks_mined_total{block="seagrass",player="ediri"} 5
minecraft_prometheus_exporter_blocks_mined_total{block="spruce_leaves",player="ediri"} 11
minecraft_prometheus_exporter_blocks_mined_total{block="spruce_log",player="ediri"} 13
minecraft_prometheus_exporter_blocks_mined_total{block="stone",player="ediri"} 2
minecraft_prometheus_exporter_blocks_mined_total{block="tall_seagrass",player="ediri"} 5
minecraft_prometheus_exporter_blocks_mined_total{block="vine",player="ediri"} 2
# HELP minecraft_prometheus_exporter_boat_one_cm_total The total distance traveled by boats.
# TYPE minecraft_prometheus_exporter_boat_one_cm_total counter
minecraft_prometheus_exporter_boat_one_cm_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_build_info A metric with a constant '1' value labeled by version, revision, branch, and goversion from which minecraft_prometheus_exporter was built.
# TYPE minecraft_prometheus_exporter_build_info gauge
minecraft_prometheus_exporter_build_info{branch="",goversion="go1.16.5",revision="",version=""} 1
# HELP minecraft_prometheus_exporter_clean_armor_total The number of dyed leather armors washed with a cauldron.
# TYPE minecraft_prometheus_exporter_clean_armor_total counter
minecraft_prometheus_exporter_clean_armor_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_clean_banner_total The number of banner patterns washed with a cauldron.
# TYPE minecraft_prometheus_exporter_clean_banner_total counter
minecraft_prometheus_exporter_clean_banner_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_clean_shulker_box_total The number of times the player has washed a Shulker Box with a cauldron.
# TYPE minecraft_prometheus_exporter_clean_shulker_box_total counter
minecraft_prometheus_exporter_clean_shulker_box_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_climb_one_cm_total The total distance traveled up ladders or vines.
# TYPE minecraft_prometheus_exporter_climb_one_cm_total counter
minecraft_prometheus_exporter_climb_one_cm_total{player="ediri"} 16
# HELP minecraft_prometheus_exporter_crouch_one_cm_total The total distance walked while sneaking.
# TYPE minecraft_prometheus_exporter_crouch_one_cm_total counter
minecraft_prometheus_exporter_crouch_one_cm_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_damage_absorbed_total The amount of damage the player has absorbed in tenths of 1♥.
# TYPE minecraft_prometheus_exporter_damage_absorbed_total counter
minecraft_prometheus_exporter_damage_absorbed_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_damage_blocked_by_shield_total The amount of damage the player has blocked with a shield in tenths of 1♥.
# TYPE minecraft_prometheus_exporter_damage_blocked_by_shield_total counter
minecraft_prometheus_exporter_damage_blocked_by_shield_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_damage_dealt_absorbed_total The amount of damage the player has dealt that were absorbed, in tenths of 1♥.
# TYPE minecraft_prometheus_exporter_damage_dealt_absorbed_total counter
minecraft_prometheus_exporter_damage_dealt_absorbed_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_damage_dealt_resisted_total The amount of damage the player has dealt that were resisted, in tenths of 1♥.
# TYPE minecraft_prometheus_exporter_damage_dealt_resisted_total counter
minecraft_prometheus_exporter_damage_dealt_resisted_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_damage_dealt_total The amount of damage the player has dealt in tenths 1♥. Includes only melee attacks.
# TYPE minecraft_prometheus_exporter_damage_dealt_total counter
minecraft_prometheus_exporter_damage_dealt_total{player="ediri"} 306
# HELP minecraft_prometheus_exporter_damage_resisted_total The amount of damage the player has resisted in tenths of 1♥.
# TYPE minecraft_prometheus_exporter_damage_resisted_total counter
minecraft_prometheus_exporter_damage_resisted_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_damage_taken_total The amount of damage the player has taken in tenths of 1♥.
# TYPE minecraft_prometheus_exporter_damage_taken_total counter
minecraft_prometheus_exporter_damage_taken_total{player="ediri"} 805
# HELP minecraft_prometheus_exporter_deaths_total The number of times the player died.
# TYPE minecraft_prometheus_exporter_deaths_total counter
minecraft_prometheus_exporter_deaths_total{player="ediri"} 3
# HELP minecraft_prometheus_exporter_eat_cake_slice_total The number of cake slices eaten.
# TYPE minecraft_prometheus_exporter_eat_cake_slice_total counter
minecraft_prometheus_exporter_eat_cake_slice_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_enchant_item_total The number of items enchanted.
# TYPE minecraft_prometheus_exporter_enchant_item_total counter
minecraft_prometheus_exporter_enchant_item_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_entities_killed_total Statistics related to the number of entities a player killed
# TYPE minecraft_prometheus_exporter_entities_killed_total counter
minecraft_prometheus_exporter_entities_killed_total{entity="pig",player="ediri"} 2
# HELP minecraft_prometheus_exporter_fall_one_cm_total The total distance fallen, excluding jumping. 
# TYPE minecraft_prometheus_exporter_fall_one_cm_total counter
minecraft_prometheus_exporter_fall_one_cm_total{player="ediri"} 2533
# HELP minecraft_prometheus_exporter_fill_cauldron_total The number of times the player filled cauldrons with water buckets.
# TYPE minecraft_prometheus_exporter_fill_cauldron_total counter
minecraft_prometheus_exporter_fill_cauldron_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_fish_caught_total The number of fish caught.
# TYPE minecraft_prometheus_exporter_fish_caught_total counter
minecraft_prometheus_exporter_fish_caught_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_fly_one_cm_total Distance traveled upwards and forwards at the same time, while more than one block above the ground.
# TYPE minecraft_prometheus_exporter_fly_one_cm_total counter
minecraft_prometheus_exporter_fly_one_cm_total{player="ediri"} 3089
# HELP minecraft_prometheus_exporter_horse_one_cm_total The total distance traveled by horses..
# TYPE minecraft_prometheus_exporter_horse_one_cm_total counter
minecraft_prometheus_exporter_horse_one_cm_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_inspect_dispenser_total The number of times interacted with dispensers.
# TYPE minecraft_prometheus_exporter_inspect_dispenser_total counter
minecraft_prometheus_exporter_inspect_dispenser_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_inspect_dropper_total The number of times interacted with droppers.
# TYPE minecraft_prometheus_exporter_inspect_dropper_total counter
minecraft_prometheus_exporter_inspect_dropper_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_inspect_hopper_total The number of times interacted with hoppers.
# TYPE minecraft_prometheus_exporter_inspect_hopper_total counter
minecraft_prometheus_exporter_inspect_hopper_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_interact_with_anvil_total The number of times interacted with anvils.
# TYPE minecraft_prometheus_exporter_interact_with_anvil_total counter
minecraft_prometheus_exporter_interact_with_anvil_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_interact_with_beacon_total The number of times interacted with beacons.
# TYPE minecraft_prometheus_exporter_interact_with_beacon_total counter
minecraft_prometheus_exporter_interact_with_beacon_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_interact_with_blast_furnace_total The number of times interacted with blast furnaces
# TYPE minecraft_prometheus_exporter_interact_with_blast_furnace_total counter
minecraft_prometheus_exporter_interact_with_blast_furnace_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_interact_with_brewingstand_total The number of times interacted with brewing stands
# TYPE minecraft_prometheus_exporter_interact_with_brewingstand_total counter
minecraft_prometheus_exporter_interact_with_brewingstand_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_interact_with_campfire_total The number of times interacted with campfires
# TYPE minecraft_prometheus_exporter_interact_with_campfire_total counter
minecraft_prometheus_exporter_interact_with_campfire_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_interact_with_cartography_table_total The number of times interacted with cartography tables
# TYPE minecraft_prometheus_exporter_interact_with_cartography_table_total counter
minecraft_prometheus_exporter_interact_with_cartography_table_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_interact_with_crafting_table_total The number of times interacted with crafting tables
# TYPE minecraft_prometheus_exporter_interact_with_crafting_table_total counter
minecraft_prometheus_exporter_interact_with_crafting_table_total{player="ediri"} 8
# HELP minecraft_prometheus_exporter_interact_with_furnace_total The number of times interacted with furnaces
# TYPE minecraft_prometheus_exporter_interact_with_furnace_total counter
minecraft_prometheus_exporter_interact_with_furnace_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_interact_with_grindstone_total The number of times interacted with grindstones
# TYPE minecraft_prometheus_exporter_interact_with_grindstone_total counter
minecraft_prometheus_exporter_interact_with_grindstone_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_interact_with_lectern_total The number of times interacted with lecterns
# TYPE minecraft_prometheus_exporter_interact_with_lectern_total counter
minecraft_prometheus_exporter_interact_with_lectern_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_interact_with_loom_total The number of times interacted with looms
# TYPE minecraft_prometheus_exporter_interact_with_loom_total counter
minecraft_prometheus_exporter_interact_with_loom_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_interact_with_smithing_table_total The number of times interacted with smithing tables.
# TYPE minecraft_prometheus_exporter_interact_with_smithing_table_total counter
minecraft_prometheus_exporter_interact_with_smithing_table_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_interact_with_smoker_total The number of times interacted with smokers.
# TYPE minecraft_prometheus_exporter_interact_with_smoker_total counter
minecraft_prometheus_exporter_interact_with_smoker_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_interact_with_stonecutter_total The number of times interacted with stonecutters.
# TYPE minecraft_prometheus_exporter_interact_with_stonecutter_total counter
minecraft_prometheus_exporter_interact_with_stonecutter_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_item_broken_total Statistics related to the number of items a player ran their durability negative
# TYPE minecraft_prometheus_exporter_item_broken_total counter
minecraft_prometheus_exporter_item_broken_total{entity="wooden_axe",player="ediri"} 1
# HELP minecraft_prometheus_exporter_item_crafted_total Statistics related to the number of items crafted, smelted, etc.
# TYPE minecraft_prometheus_exporter_item_crafted_total counter
minecraft_prometheus_exporter_item_crafted_total{block="crafting_table",player="ediri"} 2
minecraft_prometheus_exporter_item_crafted_total{block="oak_planks",player="ediri"} 16
minecraft_prometheus_exporter_item_crafted_total{block="oak_wood",player="ediri"} 3
minecraft_prometheus_exporter_item_crafted_total{block="spruce_planks",player="ediri"} 28
minecraft_prometheus_exporter_item_crafted_total{block="stick",player="ediri"} 16
minecraft_prometheus_exporter_item_crafted_total{block="torch",player="ediri"} 8
minecraft_prometheus_exporter_item_crafted_total{block="wooden_axe",player="ediri"} 1
minecraft_prometheus_exporter_item_crafted_total{block="wooden_pickaxe",player="ediri"} 2
minecraft_prometheus_exporter_item_crafted_total{block="wooden_shovel",player="ediri"} 1
# HELP minecraft_prometheus_exporter_item_dropped_total Statistics related to the number of items that droped.
# TYPE minecraft_prometheus_exporter_item_dropped_total counter
minecraft_prometheus_exporter_item_dropped_total{entity="porkchop",player="ediri"} 4
# HELP minecraft_prometheus_exporter_item_picked_up_total Statistics related to the number of dropped items a player picked up
# TYPE minecraft_prometheus_exporter_item_picked_up_total counter
minecraft_prometheus_exporter_item_picked_up_total{entity="coal",player="ediri"} 12
minecraft_prometheus_exporter_item_picked_up_total{entity="crafting_table",player="ediri"} 1
minecraft_prometheus_exporter_item_picked_up_total{entity="dirt",player="ediri"} 32
minecraft_prometheus_exporter_item_picked_up_total{entity="oak_log",player="ediri"} 5
minecraft_prometheus_exporter_item_picked_up_total{entity="oxeye_daisy",player="ediri"} 1
minecraft_prometheus_exporter_item_picked_up_total{entity="porkchop",player="ediri"} 4
minecraft_prometheus_exporter_item_picked_up_total{entity="spruce_log",player="ediri"} 13
minecraft_prometheus_exporter_item_picked_up_total{entity="wheat_seeds",player="ediri"} 5
# HELP minecraft_prometheus_exporter_item_used_total Statistics related to the number of block or item used
# TYPE minecraft_prometheus_exporter_item_used_total counter
minecraft_prometheus_exporter_item_used_total{entity="crafting_table",player="ediri"} 2
minecraft_prometheus_exporter_item_used_total{entity="oxeye_daisy",player="ediri"} 1
minecraft_prometheus_exporter_item_used_total{entity="torch",player="ediri"} 1
minecraft_prometheus_exporter_item_used_total{entity="wooden_axe",player="ediri"} 52
minecraft_prometheus_exporter_item_used_total{entity="wooden_pickaxe",player="ediri"} 22
# HELP minecraft_prometheus_exporter_items_drop_total The number of items dropped.
# TYPE minecraft_prometheus_exporter_items_drop_total counter
minecraft_prometheus_exporter_items_drop_total{player="ediri"} 1
# HELP minecraft_prometheus_exporter_jump_total         The total number of jumps performed.
# TYPE minecraft_prometheus_exporter_jump_total counter
minecraft_prometheus_exporter_jump_total{player="ediri"} 239
# HELP minecraft_prometheus_exporter_killed_by_total Statistics related to the times of a player being killed by entities.
# TYPE minecraft_prometheus_exporter_killed_by_total counter
minecraft_prometheus_exporter_killed_by_total{entity="zombie",player="ediri"} 2
# HELP minecraft_prometheus_exporter_leave_game_total The number of times "Save and quit to title" has been clicked.
# TYPE minecraft_prometheus_exporter_leave_game_total counter
minecraft_prometheus_exporter_leave_game_total{player="ediri"} 8
# HELP minecraft_prometheus_exporter_minecart_one_cm_total The total distance traveled by minecarts.
# TYPE minecraft_prometheus_exporter_minecart_one_cm_total counter
minecraft_prometheus_exporter_minecart_one_cm_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_mob_kills_total The number of mobs the player killed.
# TYPE minecraft_prometheus_exporter_mob_kills_total counter
minecraft_prometheus_exporter_mob_kills_total{player="ediri"} 2
# HELP minecraft_prometheus_exporter_open_barrel_total The number of times the player has opened a Barrel.
# TYPE minecraft_prometheus_exporter_open_barrel_total counter
minecraft_prometheus_exporter_open_barrel_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_open_chest_total The number of times the player opened chests.
# TYPE minecraft_prometheus_exporter_open_chest_total counter
minecraft_prometheus_exporter_open_chest_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_open_enderchest_total The number of times the player opened ender chests.
# TYPE minecraft_prometheus_exporter_open_enderchest_total counter
minecraft_prometheus_exporter_open_enderchest_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_open_shulker_box_total The number of times the player has opened a Shulker Box.
# TYPE minecraft_prometheus_exporter_open_shulker_box_total counter
minecraft_prometheus_exporter_open_shulker_box_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_pig_one_cm_total The total distance traveled by pigs via saddles.
# TYPE minecraft_prometheus_exporter_pig_one_cm_total counter
minecraft_prometheus_exporter_pig_one_cm_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_play_noteblock_total The number of note blocks hit.
# TYPE minecraft_prometheus_exporter_play_noteblock_total counter
minecraft_prometheus_exporter_play_noteblock_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_play_record_total The number of music discs played on a jukebox.
# TYPE minecraft_prometheus_exporter_play_record_total counter
minecraft_prometheus_exporter_play_record_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_play_time_total The total amount of time played. 
# TYPE minecraft_prometheus_exporter_play_time_total counter
minecraft_prometheus_exporter_play_time_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_player_advancements_total Number of completed advances of a player
# TYPE minecraft_prometheus_exporter_player_advancements_total counter
minecraft_prometheus_exporter_player_advancements_total{player="ediri"} 51
# HELP minecraft_prometheus_exporter_player_current_xp_total How much current XP a player has
# TYPE minecraft_prometheus_exporter_player_current_xp_total counter
minecraft_prometheus_exporter_player_current_xp_total{player="ediri"} 2
# HELP minecraft_prometheus_exporter_player_food_level_total How much food the player currently has
# TYPE minecraft_prometheus_exporter_player_food_level_total counter
minecraft_prometheus_exporter_player_food_level_total{player="ediri"} 17
# HELP minecraft_prometheus_exporter_player_health_total How much Health the player currently has
# TYPE minecraft_prometheus_exporter_player_health_total counter
minecraft_prometheus_exporter_player_health_total{player="ediri"} 19
# HELP minecraft_prometheus_exporter_player_kills_total The number of players the player killed
# TYPE minecraft_prometheus_exporter_player_kills_total counter
minecraft_prometheus_exporter_player_kills_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_player_score_total The Score of the player
# TYPE minecraft_prometheus_exporter_player_score_total counter
minecraft_prometheus_exporter_player_score_total{player="ediri"} 18
# HELP minecraft_prometheus_exporter_player_xp_total How much total XP a player has
# TYPE minecraft_prometheus_exporter_player_xp_total counter
minecraft_prometheus_exporter_player_xp_total{player="ediri"} 18
# HELP minecraft_prometheus_exporter_pot_flower_total The number of plants potted onto flower pots.
# TYPE minecraft_prometheus_exporter_pot_flower_total counter
minecraft_prometheus_exporter_pot_flower_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_raid_trigger_total The number of times the player has triggered a Raid.
# TYPE minecraft_prometheus_exporter_raid_trigger_total counter
minecraft_prometheus_exporter_raid_trigger_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_raid_win_total The number of times the player has won a Raid.
# TYPE minecraft_prometheus_exporter_raid_win_total counter
minecraft_prometheus_exporter_raid_win_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_sleep_in_bed_total The number of times the player has slept in a bed..
# TYPE minecraft_prometheus_exporter_sleep_in_bed_total counter
minecraft_prometheus_exporter_sleep_in_bed_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_sneak_time_total The time the player has held down the sneak button.
# TYPE minecraft_prometheus_exporter_sneak_time_total counter
minecraft_prometheus_exporter_sneak_time_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_sprint_one_cm_total The total distance sprinted.
# TYPE minecraft_prometheus_exporter_sprint_one_cm_total counter
minecraft_prometheus_exporter_sprint_one_cm_total{player="ediri"} 17
# HELP minecraft_prometheus_exporter_strider_one_cm_total The total distance traveled by striders via saddles.
# TYPE minecraft_prometheus_exporter_strider_one_cm_total counter
minecraft_prometheus_exporter_strider_one_cm_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_swim_one_cm_total The total distance covered with sprint-swimming..
# TYPE minecraft_prometheus_exporter_swim_one_cm_total counter
minecraft_prometheus_exporter_swim_one_cm_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_talked_to_villager_total The number of times interacted with villagers (opened the trading GUI).
# TYPE minecraft_prometheus_exporter_talked_to_villager_total counter
minecraft_prometheus_exporter_talked_to_villager_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_target_hit_total The number of times the player has shot a target block.
# TYPE minecraft_prometheus_exporter_target_hit_total counter
minecraft_prometheus_exporter_target_hit_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_time_since_death_total The time since the player's last death.
# TYPE minecraft_prometheus_exporter_time_since_death_total counter
minecraft_prometheus_exporter_time_since_death_total{player="ediri"} 19806
# HELP minecraft_prometheus_exporter_time_since_rest_total The time since the player's last rest. This is used to spawn phantoms.
# TYPE minecraft_prometheus_exporter_time_since_rest_total counter
minecraft_prometheus_exporter_time_since_rest_total{player="ediri"} 20383
# HELP minecraft_prometheus_exporter_total_world_time_total The total amount of time the world was opened.n.
# TYPE minecraft_prometheus_exporter_total_world_time_total counter
minecraft_prometheus_exporter_total_world_time_total{player="ediri"} 66046
# HELP minecraft_prometheus_exporter_traded_with_villager_total The number of times traded with villagers.
# TYPE minecraft_prometheus_exporter_traded_with_villager_total counter
minecraft_prometheus_exporter_traded_with_villager_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_trigger_trapped_chest_total The number of times the player opened trapped chests.
# TYPE minecraft_prometheus_exporter_trigger_trapped_chest_total counter
minecraft_prometheus_exporter_trigger_trapped_chest_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_tune_noteblock_total The number of times interacted with note blocks.
# TYPE minecraft_prometheus_exporter_tune_noteblock_total counter
minecraft_prometheus_exporter_tune_noteblock_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_use_cauldron_total The number of times the player took water from cauldrons with glass bottles.
# TYPE minecraft_prometheus_exporter_use_cauldron_total counter
minecraft_prometheus_exporter_use_cauldron_total{player="ediri"} 0
# HELP minecraft_prometheus_exporter_walk_on_water_one_cm_total The distance covered while bobbing up and down over water.
# TYPE minecraft_prometheus_exporter_walk_on_water_one_cm_total counter
minecraft_prometheus_exporter_walk_on_water_one_cm_total{player="ediri"} 11460
# HELP minecraft_prometheus_exporter_walk_one_cm_total The total distance walked.
# TYPE minecraft_prometheus_exporter_walk_one_cm_total counter
minecraft_prometheus_exporter_walk_one_cm_total{player="ediri"} 78188
# HELP minecraft_prometheus_exporter_walk_under_water_one_cm_total The total distance you have walked underwater.
# TYPE minecraft_prometheus_exporter_walk_under_water_one_cm_total counter
minecraft_prometheus_exporter_walk_under_water_one_cm_total{player="ediri"} 14486

```

RichiH commented:
Thanks for this. How many players are a common maximum; should we be worried about cardinality explosion?

Some comments/nits re naming:
* Renaming `minecraft_prometheus_exporter_*` to `minecraft_*` would be more customary
* Rename `interact_with` -> `interactions`?
* Is world time in seconds? `minecraft_prometheus_exporter_total_world_time_total` -> `*_seconds_total` would be better. Same for other time.
* `minecraft_prometheus_exporter_open_chest_total` -> `minecraft_prometheus_exporter_opened_total{type="chest"}` etc.?
* `minecraft_prometheus_exporter_swim_one_cm_total` -> `minecraft_movement_meter_total{type="swim"}` etc.? Note that I changed to `meter` as that's the SI base unit.

dirien commented:
@roidelapluie , yes only vanilla for the moment and only java version. As Bedrock dont have rcon and stats AFAIK

dirien commented:
@RichiH: thanks for the input, i will check the renaming. Some of them require some logic changes. 

Regarding the max size of the user. Guess depends the spec of the server...

![image](https://user-images.githubusercontent.com/38325136/125457954-ccef1c6a-27cb-4af8-8a36-97a5e0c2961b.png)

dirien commented:
hi @RichiH, i updated everything with your feedback! (thanks again 👍🏻 ). The time related metric i could not change, as its counted in Ticks in minecraft. 

here is the new list, i let some examples inside.

```bash
# HELP minecraft_animals_bred_total The number of times the player bred two mobs.
# TYPE minecraft_animals_bred_total counter
minecraft_animals_bred_total{player="ediri"} 0
# HELP minecraft_bell_ring_total The number of times the player has rung a Bell.
# TYPE minecraft_bell_ring_total counter
minecraft_bell_ring_total{player="ediri"} 0
# HELP minecraft_blocks_mined_total Statistic related to the number of blocks a player mined
# TYPE minecraft_blocks_mined_total counter
minecraft_blocks_mined_total{block="birch_log",player="ediri"} 2
minecraft_blocks_mined_total{block="clay",player="ediri"} 1
minecraft_blocks_mined_total{block="coal_ore",player="ediri"} 16
minecraft_blocks_mined_total{block="crafting_table",player="ediri"} 2
minecraft_blocks_mined_total{block="dirt",player="ediri"} 19
minecraft_blocks_mined_total{block="grass",player="ediri"} 38
minecraft_blocks_mined_total{block="grass_block",player="ediri"} 18
minecraft_blocks_mined_total{block="oak_leaves",player="ediri"} 5
minecraft_blocks_mined_total{block="oak_log",player="ediri"} 11
minecraft_blocks_mined_total{block="oak_planks",player="ediri"} 1
minecraft_blocks_mined_total{block="oxeye_daisy",player="ediri"} 1
minecraft_blocks_mined_total{block="seagrass",player="ediri"} 5
minecraft_blocks_mined_total{block="spruce_leaves",player="ediri"} 11
minecraft_blocks_mined_total{block="spruce_log",player="ediri"} 13
minecraft_blocks_mined_total{block="stone",player="ediri"} 2
minecraft_blocks_mined_total{block="tall_grass",player="ediri"} 1
minecraft_blocks_mined_total{block="tall_seagrass",player="ediri"} 5
minecraft_blocks_mined_total{block="vine",player="ediri"} 2
minecraft_blocks_mined_total{block="wall_torch",player="ediri"} 1
# HELP minecraft_clean_armor_total The number of dyed leather armors washed with a cauldron.
# TYPE minecraft_clean_armor_total counter
minecraft_clean_armor_total{player="ediri"} 0
# HELP minecraft_clean_banner_total The number of banner patterns washed with a cauldron.
# TYPE minecraft_clean_banner_total counter
minecraft_clean_banner_total{player="ediri"} 0
# HELP minecraft_clean_shulker_box_total The number of times the player has washed a Shulker Box with a cauldron.
# TYPE minecraft_clean_shulker_box_total counter
minecraft_clean_shulker_box_total{player="ediri"} 0
# HELP minecraft_damage_total The amount of damage the player has handled from different types in tenths of 1♥.
# TYPE minecraft_damage_total counter
minecraft_damage_total{player="ediri",type="absorbed"} 0
minecraft_damage_total{player="ediri",type="blocked_by_shield"} 0
minecraft_damage_total{player="ediri",type="dealt"} 610
minecraft_damage_total{player="ediri",type="dealt_absorbed"} 0
minecraft_damage_total{player="ediri",type="dealt_resisted"} 0
minecraft_damage_total{player="ediri",type="resisted"} 0
minecraft_damage_total{player="ediri",type="taken"} 2295
# HELP minecraft_deaths_total The number of times the player died.
# TYPE minecraft_deaths_total counter
minecraft_deaths_total{player="ediri"} 9
# HELP minecraft_eat_cake_slice_total The number of cake slices eaten.
# TYPE minecraft_eat_cake_slice_total counter
minecraft_eat_cake_slice_total{player="ediri"} 0
# HELP minecraft_enchant_item_total The number of items enchanted.
# TYPE minecraft_enchant_item_total counter
minecraft_enchant_item_total{player="ediri"} 0
# HELP minecraft_entities_killed_total Statistics related to the number of entities a player killed
# TYPE minecraft_entities_killed_total counter
minecraft_entities_killed_total{entity="pig",player="ediri"} 2
# HELP minecraft_exporter_build_info A metric with a constant '1' value labeled by version, revision, branch, and goversion from which minecraft_exporter was built.
# TYPE minecraft_exporter_build_info gauge
minecraft_exporter_build_info{branch="",goversion="go1.16.5",revision="",version=""} 1
# HELP minecraft_fill_cauldron_total The number of times the player filled cauldrons with water buckets.
# TYPE minecraft_fill_cauldron_total counter
minecraft_fill_cauldron_total{player="ediri"} 0
# HELP minecraft_fish_caught_total The number of fish caught.
# TYPE minecraft_fish_caught_total counter
minecraft_fish_caught_total{player="ediri"} 0
# HELP minecraft_inspected_total The number of times inspected a dispenser, hopper or dropper.
# TYPE minecraft_inspected_total counter
minecraft_inspected_total{entity="dispenser",player="ediri"} 0
minecraft_inspected_total{entity="dropper",player="ediri"} 0
minecraft_inspected_total{entity="hopper",player="ediri"} 0
# HELP minecraft_interactions_total The number of times interacted with different entities
# TYPE minecraft_interactions_total counter
minecraft_interactions_total{entity="anvil",player="ediri"} 0
minecraft_interactions_total{entity="beacon",player="ediri"} 0
minecraft_interactions_total{entity="blast_furnace",player="ediri"} 0
minecraft_interactions_total{entity="brewingstand",player="ediri"} 0
minecraft_interactions_total{entity="campfire",player="ediri"} 0
minecraft_interactions_total{entity="cartography_table",player="ediri"} 0
minecraft_interactions_total{entity="crafting_table",player="ediri"} 10
minecraft_interactions_total{entity="furnace",player="ediri"} 0
minecraft_interactions_total{entity="grindston",player="ediri"} 0
minecraft_interactions_total{entity="lectern",player="ediri"} 0
minecraft_interactions_total{entity="loom",player="ediri"} 0
minecraft_interactions_total{entity="smithing_table",player="ediri"} 0
minecraft_interactions_total{entity="smoker",player="ediri"} 0
minecraft_interactions_total{entity="stonecutter",player="ediri"} 0
# HELP minecraft_item_actions_total Statistics related to the number of items and their actions: used, picked up, dropped, broken
# TYPE minecraft_item_actions_total counter
minecraft_item_actions_total{action="broken",entity="wooden_axe",player="ediri"} 1
minecraft_item_actions_total{action="crafted",entity="crafting_table",player="ediri"} 3
minecraft_item_actions_total{action="crafted",entity="oak_planks",player="ediri"} 32
minecraft_item_actions_total{action="crafted",entity="oak_wood",player="ediri"} 6
minecraft_item_actions_total{action="crafted",entity="spruce_planks",player="ediri"} 28
minecraft_item_actions_total{action="crafted",entity="stick",player="ediri"} 32
minecraft_item_actions_total{action="crafted",entity="torch",player="ediri"} 8
minecraft_item_actions_total{action="crafted",entity="wooden_axe",player="ediri"} 1
minecraft_item_actions_total{action="crafted",entity="wooden_pickaxe",player="ediri"} 3
minecraft_item_actions_total{action="crafted",entity="wooden_shovel",player="ediri"} 1
minecraft_item_actions_total{action="dropped",entity="porkchop",player="ediri"} 4
minecraft_item_actions_total{action="picked_up",entity="birch_log",player="ediri"} 1
minecraft_item_actions_total{action="picked_up",entity="coal",player="ediri"} 15
minecraft_item_actions_total{action="picked_up",entity="crafting_table",player="ediri"} 2
minecraft_item_actions_total{action="picked_up",entity="dirt",player="ediri"} 33
minecraft_item_actions_total{action="picked_up",entity="oak_log",player="ediri"} 11
minecraft_item_actions_total{action="picked_up",entity="oak_planks",player="ediri"} 1
minecraft_item_actions_total{action="picked_up",entity="oak_sapling",player="ediri"} 1
minecraft_item_actions_total{action="picked_up",entity="oxeye_daisy",player="ediri"} 1
minecraft_item_actions_total{action="picked_up",entity="porkchop",player="ediri"} 4
minecraft_item_actions_total{action="picked_up",entity="spruce_log",player="ediri"} 13
minecraft_item_actions_total{action="picked_up",entity="torch",player="ediri"} 1
minecraft_item_actions_total{action="picked_up",entity="wheat_seeds",player="ediri"} 6
minecraft_item_actions_total{action="used",entity="crafting_table",player="ediri"} 3
minecraft_item_actions_total{action="used",entity="oak_planks",player="ediri"} 1
minecraft_item_actions_total{action="used",entity="oxeye_daisy",player="ediri"} 1
minecraft_item_actions_total{action="used",entity="torch",player="ediri"} 2
minecraft_item_actions_total{action="used",entity="wooden_axe",player="ediri"} 52
minecraft_item_actions_total{action="used",entity="wooden_pickaxe",player="ediri"} 26
# HELP minecraft_items_drop_total The number of items dropped.
# TYPE minecraft_items_drop_total counter
minecraft_items_drop_total{player="ediri"} 1
# HELP minecraft_jump_total     The total number of jumps performed.
# TYPE minecraft_jump_total counter
minecraft_jump_total{player="ediri"} 406
# HELP minecraft_killed_by_total Statistics related to the times of a player being killed by entities.
# TYPE minecraft_killed_by_total counter
minecraft_killed_by_total{entity="skeleton",player="ediri"} 1
minecraft_killed_by_total{entity="zombie",player="ediri"} 7
# HELP minecraft_leave_game_total The number of times "Save and quit to title" has been clicked.
# TYPE minecraft_leave_game_total counter
minecraft_leave_game_total{player="ediri"} 11
# HELP minecraft_mob_kills_total The number of mobs the player killed.
# TYPE minecraft_mob_kills_total counter
minecraft_mob_kills_total{player="ediri"} 2
# HELP minecraft_movement_meter_total The total distance traveled with different entities (ladders, boats, etc.)
# TYPE minecraft_movement_meter_total untyped
minecraft_movement_meter_total{entity="aviate",player="ediri"} 0
minecraft_movement_meter_total{entity="boat",player="ediri"} 0
minecraft_movement_meter_total{entity="climb",player="ediri"} 0.16
minecraft_movement_meter_total{entity="crouch",player="ediri"} 0
minecraft_movement_meter_total{entity="fall",player="ediri"} 58.49
minecraft_movement_meter_total{entity="fly",player="ediri"} 55.14
minecraft_movement_meter_total{entity="horse",player="ediri"} 0
minecraft_movement_meter_total{entity="minecart",player="ediri"} 0
minecraft_movement_meter_total{entity="pig",player="ediri"} 0
minecraft_movement_meter_total{entity="sprint",player="ediri"} 0.17
minecraft_movement_meter_total{entity="strider",player="ediri"} 0
minecraft_movement_meter_total{entity="swim",player="ediri"} 0
minecraft_movement_meter_total{entity="walk",player="ediri"} 1411.7
minecraft_movement_meter_total{entity="walk_on_water",player="ediri"} 168.45
minecraft_movement_meter_total{entity="walk_under_water",player="ediri"} 156.92
# HELP minecraft_open_barrel_total The number of times the player has opened a Barrel.
# TYPE minecraft_open_barrel_total counter
minecraft_open_barrel_total{player="ediri"} 0
# HELP minecraft_open_chest_total The number of times the player opened chests.
# TYPE minecraft_open_chest_total counter
minecraft_open_chest_total{player="ediri"} 0
# HELP minecraft_open_enderchest_total The number of times the player opened ender chests.
# TYPE minecraft_open_enderchest_total counter
minecraft_open_enderchest_total{player="ediri"} 0
# HELP minecraft_open_shulker_box_total The number of times the player has opened a Shulker Box.
# TYPE minecraft_open_shulker_box_total counter
minecraft_open_shulker_box_total{player="ediri"} 0
# HELP minecraft_play_noteblock_total The number of note blocks hit.
# TYPE minecraft_play_noteblock_total counter
minecraft_play_noteblock_total{player="ediri"} 0
# HELP minecraft_play_record_total The number of music discs played on a jukebox.
# TYPE minecraft_play_record_total counter
minecraft_play_record_total{player="ediri"} 0
# HELP minecraft_play_time_total The total amount of time played. 
# TYPE minecraft_play_time_total counter
minecraft_play_time_total{player="ediri"} 0
# HELP minecraft_player_kills_total The number of players the player killed
# TYPE minecraft_player_kills_total counter
minecraft_player_kills_total{player="ediri"} 0
# HELP minecraft_player_stat_total Different stats of the player: xp, current_xp, food_level, health, score, advancements
# TYPE minecraft_player_stat_total counter
minecraft_player_stat_total{player="ediri",stat="advancements"} 53
minecraft_player_stat_total{player="ediri",stat="current_xp"} 0
minecraft_player_stat_total{player="ediri",stat="food_level"} 20
minecraft_player_stat_total{player="ediri",stat="health"} 20
minecraft_player_stat_total{player="ediri",stat="score"} 1
minecraft_player_stat_total{player="ediri",stat="xp"} 1
# HELP minecraft_pot_flower_total The number of plants potted onto flower pots.
# TYPE minecraft_pot_flower_total counter
minecraft_pot_flower_total{player="ediri"} 0
# HELP minecraft_raid_trigger_total The number of times the player has triggered a Raid.
# TYPE minecraft_raid_trigger_total counter
minecraft_raid_trigger_total{player="ediri"} 0
# HELP minecraft_raid_win_total The number of times the player has won a Raid.
# TYPE minecraft_raid_win_total counter
minecraft_raid_win_total{player="ediri"} 0
# HELP minecraft_sleep_in_bed_total The number of times the player has slept in a bed..
# TYPE minecraft_sleep_in_bed_total counter
minecraft_sleep_in_bed_total{player="ediri"} 0
# HELP minecraft_sneak_time_total The time the player has held down the sneak button.
# TYPE minecraft_sneak_time_total counter
minecraft_sneak_time_total{player="ediri"} 0
# HELP minecraft_talked_to_villager_total The number of times interacted with villagers (opened the trading GUI).
# TYPE minecraft_talked_to_villager_total counter
minecraft_talked_to_villager_total{player="ediri"} 0
# HELP minecraft_target_hit_total The number of times the player has shot a target block.
# TYPE minecraft_target_hit_total counter
minecraft_target_hit_total{player="ediri"} 0
# HELP minecraft_time_since_death_total The time since the player's last death.
# TYPE minecraft_time_since_death_total counter
minecraft_time_since_death_total{player="ediri"} 3373
# HELP minecraft_time_since_rest_total The time since the player's last rest. This is used to spawn phantoms.
# TYPE minecraft_time_since_rest_total counter
minecraft_time_since_rest_total{player="ediri"} 3394
# HELP minecraft_total_world_time_total The total amount of time the world was opened.n.
# TYPE minecraft_total_world_time_total counter
minecraft_total_world_time_total{player="ediri"} 138851
# HELP minecraft_traded_with_villager_total The number of times traded with villagers.
# TYPE minecraft_traded_with_villager_total counter
minecraft_traded_with_villager_total{player="ediri"} 0
# HELP minecraft_trigger_trapped_chest_total The number of times the player opened trapped chests.
# TYPE minecraft_trigger_trapped_chest_total counter
minecraft_trigger_trapped_chest_total{player="ediri"} 0
# HELP minecraft_tune_noteblock_total The number of times interacted with note blocks.
# TYPE minecraft_tune_noteblock_total counter
minecraft_tune_noteblock_total{player="ediri"} 0
# HELP minecraft_use_cauldron_total The number of times the player took water from cauldrons with glass bottles.
# TYPE minecraft_use_cauldron_total counter
minecraft_use_cauldron_total{player="ediri"} 0
```

RichiH commented:
Thanks!

I think `minecraft_play_time_ticks_total` would make sense in this case.

There's still a bunch of singular words and such, e.g. `minecraft_open_barrel_total` instead of `minecraft_openED_barrelS_total`

Are `minecraft_player_stat_total` food level & health counters or gauges?

dirien commented:
@RichiH , good point with the personal stats. They should be gauge instead counter as they go in both sides. The others stats are counter as they only go up in value.

with the ticks i was not sure if ticks is a valid base unit. Like mentioned https://prometheus.io/docs/practices/naming/#base-units here.

I will check the plurals too.

Thanks for the super feedback!

RichiH commented:
There are many things which are not SI base units and still countable; in those cases using what makes sense is best.

Also thanks to you for the quick reactions :)

dirien commented:
So now :) i have a good feeling, that you will like it:

```bash
# HELP minecraft_animals_breded_total The number of times the player bred two mobs.
# TYPE minecraft_animals_breded_total counter
minecraft_animals_breded_total{player="ediri"} 0
# HELP minecraft_bells_ringed_total The number of times the player has rung a Bell.
# TYPE minecraft_bells_ringed_total counter
minecraft_bells_ringed_total{player="ediri"} 0
# HELP minecraft_blocks_mined_total Statistic related to the number of blocks a player mined
# TYPE minecraft_blocks_mined_total counter
minecraft_blocks_mined_total{block="birch_log",player="ediri"} 2
minecraft_blocks_mined_total{block="clay",player="ediri"} 1
minecraft_blocks_mined_total{block="coal_ore",player="ediri"} 16
minecraft_blocks_mined_total{block="crafting_table",player="ediri"} 2
minecraft_blocks_mined_total{block="dirt",player="ediri"} 19
minecraft_blocks_mined_total{block="grass",player="ediri"} 38
minecraft_blocks_mined_total{block="grass_block",player="ediri"} 18
minecraft_blocks_mined_total{block="oak_leaves",player="ediri"} 5
minecraft_blocks_mined_total{block="oak_log",player="ediri"} 11
minecraft_blocks_mined_total{block="oak_planks",player="ediri"} 1
minecraft_blocks_mined_total{block="oxeye_daisy",player="ediri"} 1
minecraft_blocks_mined_total{block="seagrass",player="ediri"} 5
minecraft_blocks_mined_total{block="spruce_leaves",player="ediri"} 11
minecraft_blocks_mined_total{block="spruce_log",player="ediri"} 13
minecraft_blocks_mined_total{block="stone",player="ediri"} 2
minecraft_blocks_mined_total{block="tall_grass",player="ediri"} 1
minecraft_blocks_mined_total{block="tall_seagrass",player="ediri"} 5
minecraft_blocks_mined_total{block="vine",player="ediri"} 2
minecraft_blocks_mined_total{block="wall_torch",player="ediri"} 1
# HELP minecraft_cake_slices_eaten_total The number of cake slices eaten.
# TYPE minecraft_cake_slices_eaten_total counter
minecraft_cake_slices_eaten_total{player="ediri"} 0
# HELP minecraft_cleaned_armors_total The number of dyed leather armors washed with a cauldron.
# TYPE minecraft_cleaned_armors_total counter
minecraft_cleaned_armors_total{player="ediri"} 0
# HELP minecraft_cleaned_banner_total The number of banner patterns washed with a cauldron.
# TYPE minecraft_cleaned_banner_total counter
minecraft_cleaned_banner_total{player="ediri"} 0
# HELP minecraft_damage_total The amount of damage the player has handled from different types in tenths of 1♥.
# TYPE minecraft_damage_total counter
minecraft_damage_total{player="ediri",type="absorbed"} 0
minecraft_damage_total{player="ediri",type="blocked_by_shield"} 0
minecraft_damage_total{player="ediri",type="dealt"} 610
minecraft_damage_total{player="ediri",type="dealt_absorbed"} 0
minecraft_damage_total{player="ediri",type="dealt_resisted"} 0
minecraft_damage_total{player="ediri",type="resisted"} 0
minecraft_damage_total{player="ediri",type="taken"} 2295
# HELP minecraft_deaths_total The number of times the player died.
# TYPE minecraft_deaths_total counter
minecraft_deaths_total{player="ediri"} 9
# HELP minecraft_entities_killed_total Statistics related to the number of entities a player killed
# TYPE minecraft_entities_killed_total counter
minecraft_entities_killed_total{entity="pig",player="ediri"} 2
# HELP minecraft_exporter_build_info A metric with a constant '1' value labeled by version, revision, branch, and goversion from which minecraft_exporter was built.
# TYPE minecraft_exporter_build_info gauge
minecraft_exporter_build_info{branch="",goversion="go1.16.5",revision="",version=""} 1
# HELP minecraft_filled_cauldrons_total The number of times the player filled cauldrons with water buckets.
# TYPE minecraft_filled_cauldrons_total counter
minecraft_filled_cauldrons_total{player="ediri"} 0
# HELP minecraft_fishs_caught_total The number of fish caught.
# TYPE minecraft_fishs_caught_total counter
minecraft_fishs_caught_total{player="ediri"} 0
# HELP minecraft_games_left_total The number of times "Save and quit to title" has been clicked.
# TYPE minecraft_games_left_total counter
minecraft_games_left_total{player="ediri"} 12
# HELP minecraft_inspected_total The number of times inspected a dispenser, hopper or dropper.
# TYPE minecraft_inspected_total counter
minecraft_inspected_total{entity="dispenser",player="ediri"} 0
minecraft_inspected_total{entity="dropper",player="ediri"} 0
minecraft_inspected_total{entity="hopper",player="ediri"} 0
# HELP minecraft_interactions_total The number of times interacted with different entities
# TYPE minecraft_interactions_total counter
minecraft_interactions_total{entity="anvil",player="ediri"} 0
minecraft_interactions_total{entity="beacon",player="ediri"} 0
minecraft_interactions_total{entity="blast_furnace",player="ediri"} 0
minecraft_interactions_total{entity="brewingstand",player="ediri"} 0
minecraft_interactions_total{entity="campfire",player="ediri"} 0
minecraft_interactions_total{entity="cartography_table",player="ediri"} 0
minecraft_interactions_total{entity="crafting_table",player="ediri"} 10
minecraft_interactions_total{entity="furnace",player="ediri"} 0
minecraft_interactions_total{entity="grindston",player="ediri"} 0
minecraft_interactions_total{entity="lectern",player="ediri"} 0
minecraft_interactions_total{entity="loom",player="ediri"} 0
minecraft_interactions_total{entity="smithing_table",player="ediri"} 0
minecraft_interactions_total{entity="smoker",player="ediri"} 0
minecraft_interactions_total{entity="stonecutter",player="ediri"} 0
# HELP minecraft_item_actions_total Statistics related to the number of items and their actions: used, picked up, dropped, broken
# TYPE minecraft_item_actions_total counter
minecraft_item_actions_total{action="broken",entity="wooden_axe",player="ediri"} 1
minecraft_item_actions_total{action="crafted",entity="crafting_table",player="ediri"} 3
minecraft_item_actions_total{action="crafted",entity="oak_planks",player="ediri"} 32
minecraft_item_actions_total{action="crafted",entity="oak_wood",player="ediri"} 6
minecraft_item_actions_total{action="crafted",entity="spruce_planks",player="ediri"} 28
minecraft_item_actions_total{action="crafted",entity="stick",player="ediri"} 32
minecraft_item_actions_total{action="crafted",entity="torch",player="ediri"} 8
minecraft_item_actions_total{action="crafted",entity="wooden_axe",player="ediri"} 1
minecraft_item_actions_total{action="crafted",entity="wooden_pickaxe",player="ediri"} 3
minecraft_item_actions_total{action="crafted",entity="wooden_shovel",player="ediri"} 1
minecraft_item_actions_total{action="dropped",entity="porkchop",player="ediri"} 4
minecraft_item_actions_total{action="picked_up",entity="birch_log",player="ediri"} 1
minecraft_item_actions_total{action="picked_up",entity="coal",player="ediri"} 15
minecraft_item_actions_total{action="picked_up",entity="crafting_table",player="ediri"} 2
minecraft_item_actions_total{action="picked_up",entity="dirt",player="ediri"} 33
minecraft_item_actions_total{action="picked_up",entity="oak_log",player="ediri"} 11
minecraft_item_actions_total{action="picked_up",entity="oak_planks",player="ediri"} 1
minecraft_item_actions_total{action="picked_up",entity="oak_sapling",player="ediri"} 1
minecraft_item_actions_total{action="picked_up",entity="oxeye_daisy",player="ediri"} 1
minecraft_item_actions_total{action="picked_up",entity="porkchop",player="ediri"} 4
minecraft_item_actions_total{action="picked_up",entity="spruce_log",player="ediri"} 13
minecraft_item_actions_total{action="picked_up",entity="torch",player="ediri"} 1
minecraft_item_actions_total{action="picked_up",entity="wheat_seeds",player="ediri"} 6
minecraft_item_actions_total{action="used",entity="crafting_table",player="ediri"} 3
minecraft_item_actions_total{action="used",entity="oak_planks",player="ediri"} 1
minecraft_item_actions_total{action="used",entity="oxeye_daisy",player="ediri"} 1
minecraft_item_actions_total{action="used",entity="torch",player="ediri"} 2
minecraft_item_actions_total{action="used",entity="wooden_axe",player="ediri"} 52
minecraft_item_actions_total{action="used",entity="wooden_pickaxe",player="ediri"} 26
# HELP minecraft_items_dropped_total The number of items dropped.
# TYPE minecraft_items_dropped_total counter
minecraft_items_dropped_total{player="ediri"} 1
# HELP minecraft_items_enchanted_total The number of items enchanted.
# TYPE minecraft_items_enchanted_total counter
minecraft_items_enchanted_total{player="ediri"} 0
# HELP minecraft_jumps_total    The total number of jumps performed.
# TYPE minecraft_jumps_total counter
minecraft_jumps_total{player="ediri"} 406
# HELP minecraft_killed_by_total Statistics related to the times of a player being killed by entities.
# TYPE minecraft_killed_by_total counter
minecraft_killed_by_total{entity="skeleton",player="ediri"} 1
minecraft_killed_by_total{entity="zombie",player="ediri"} 7
# HELP minecraft_mobs_killed_total The number of mobs the player killed.
# TYPE minecraft_mobs_killed_total counter
minecraft_mobs_killed_total{player="ediri"} 2
# HELP minecraft_movement_meters_total The total distance traveled with different entities (ladders, boats, etc.)
# TYPE minecraft_movement_meters_total untyped
minecraft_movement_meters_total{means="aviate",player="ediri"} 0
minecraft_movement_meters_total{means="boat",player="ediri"} 0
minecraft_movement_meters_total{means="climb",player="ediri"} 0.16
minecraft_movement_meters_total{means="crouch",player="ediri"} 0
minecraft_movement_meters_total{means="fall",player="ediri"} 58.49
minecraft_movement_meters_total{means="fly",player="ediri"} 55.14
minecraft_movement_meters_total{means="horse",player="ediri"} 0
minecraft_movement_meters_total{means="minecart",player="ediri"} 0
minecraft_movement_meters_total{means="pig",player="ediri"} 0
minecraft_movement_meters_total{means="sprint",player="ediri"} 0.17
minecraft_movement_meters_total{means="strider",player="ediri"} 0
minecraft_movement_meters_total{means="swim",player="ediri"} 0
minecraft_movement_meters_total{means="walk",player="ediri"} 1411.7
minecraft_movement_meters_total{means="walk_on_water",player="ediri"} 168.45
minecraft_movement_meters_total{means="walk_under_water",player="ediri"} 156.92
# HELP minecraft_noteblocks_played_total The number of note blocks hit.
# TYPE minecraft_noteblocks_played_total counter
minecraft_noteblocks_played_total{player="ediri"} 0
# HELP minecraft_noteblocks_tuned_total The number of times interacted with note blocks.
# TYPE minecraft_noteblocks_tuned_total counter
minecraft_noteblocks_tuned_total{player="ediri"} 0
# HELP minecraft_opened_barrels_total The number of times the player has opened a Barrel.
# TYPE minecraft_opened_barrels_total counter
minecraft_opened_barrels_total{player="ediri"} 0
# HELP minecraft_opened_chests_total The number of times the player opened chests.
# TYPE minecraft_opened_chests_total counter
minecraft_opened_chests_total{player="ediri"} 0
# HELP minecraft_opened_enderchests_total The number of times the player opened ender chests.
# TYPE minecraft_opened_enderchests_total counter
minecraft_opened_enderchests_total{player="ediri"} 0
# HELP minecraft_play_time_ticks_total The total amount of time played. 
# TYPE minecraft_play_time_ticks_total counter
minecraft_play_time_ticks_total{player="ediri"} 0
# HELP minecraft_player_online_total is 1 if player is online
# TYPE minecraft_player_online_total counter
minecraft_player_online_total{player="ediri"} 1
# HELP minecraft_player_stat_total Different stats of the player: xp, current_xp, food_level, health, score, advancements
# TYPE minecraft_player_stat_total gauge
minecraft_player_stat_total{player="ediri",stat="advancements"} 53
minecraft_player_stat_total{player="ediri",stat="current_xp"} 0
minecraft_player_stat_total{player="ediri",stat="food_level"} 20
minecraft_player_stat_total{player="ediri",stat="health"} 20
minecraft_player_stat_total{player="ediri",stat="score"} 1
minecraft_player_stat_total{player="ediri",stat="xp"} 1
# HELP minecraft_players_killed_total The number of players the player killed
# TYPE minecraft_players_killed_total counter
minecraft_players_killed_total{player="ediri"} 0
# HELP minecraft_pots_flowered_total The number of plants potted onto flower pots.
# TYPE minecraft_pots_flowered_total counter
minecraft_pots_flowered_total{player="ediri"} 0
# HELP minecraft_records_played_total The number of music discs played on a jukebox.
# TYPE minecraft_records_played_total counter
minecraft_records_played_total{player="ediri"} 0
# HELP minecraft_shulker_boxes_cleaned_total The number of times the player has washed a Shulker Box with a cauldron.
# TYPE minecraft_shulker_boxes_cleaned_total counter
minecraft_shulker_boxes_cleaned_total{player="ediri"} 0
# HELP minecraft_shulker_boxes_opened_total The number of times the player has opened a Shulker Box.
# TYPE minecraft_shulker_boxes_opened_total counter
minecraft_shulker_boxes_opened_total{player="ediri"} 0
# HELP minecraft_sleep_in_bed_ticks_total The number of times the player has slept in a bed..
# TYPE minecraft_sleep_in_bed_ticks_total counter
minecraft_sleep_in_bed_ticks_total{player="ediri"} 0
# HELP minecraft_sneak_time_ticks_total The time the player has held down the sneak button.
# TYPE minecraft_sneak_time_ticks_total counter
minecraft_sneak_time_ticks_total{player="ediri"} 0
# HELP minecraft_talked_to_villagers_total The number of times interacted with villagers (opened the trading GUI).
# TYPE minecraft_talked_to_villagers_total counter
minecraft_talked_to_villagers_total{player="ediri"} 0
# HELP minecraft_targets_hit_total The number of times the player has shot a target block.
# TYPE minecraft_targets_hit_total counter
minecraft_targets_hit_total{player="ediri"} 0
# HELP minecraft_time_since_death_ticks_total The time since the player's last death.
# TYPE minecraft_time_since_death_ticks_total counter
minecraft_time_since_death_ticks_total{player="ediri"} 24217
# HELP minecraft_time_since_rest_ticks_total The time since the player's last rest. This is used to spawn phantoms.
# TYPE minecraft_time_since_rest_ticks_total counter
minecraft_time_since_rest_ticks_total{player="ediri"} 24238
# HELP minecraft_total_world_time_ticks_total The total amount of time the world was opened.n.
# TYPE minecraft_total_world_time_ticks_total counter
minecraft_total_world_time_ticks_total{player="ediri"} 159695
# HELP minecraft_traded_with_villagers_total The number of times traded with villagers.
# TYPE minecraft_traded_with_villagers_total counter
minecraft_traded_with_villagers_total{player="ediri"} 0
# HELP minecraft_triggered_raids_total The number of times the player has triggered a Raid.
# TYPE minecraft_triggered_raids_total counter
minecraft_triggered_raids_total{player="ediri"} 0
# HELP minecraft_triggered_trapped_chests_total The number of times the player opened trapped chests.
# TYPE minecraft_triggered_trapped_chests_total counter
minecraft_triggered_trapped_chests_total{player="ediri"} 0
# HELP minecraft_used_cauldrons_total The number of times the player took water from cauldrons with glass bottles.
# TYPE minecraft_used_cauldrons_total counter
minecraft_used_cauldrons_total{player="ediri"} 0
# HELP minecraft_won_raids_total The number of times the player has won a Raid.
# TYPE minecraft_won_raids_total counter
minecraft_won_raids_total{player="ediri"} 0
```

RichiH commented:
Thanks! Sorry, I missed one: `minecraft_damage_total` should be split in received and dealt in the metric name, not via labels.

Given your speed, I will go ahead and merge already.

dirien commented:
Hi @RichiH ,

thanks for everything! Learned really much about the details of an exporter! 

Here the last comment, fixed:

```bash
# HELP minecraft_damage_dealt_total The amount of damage the player has dealt from different types in tenths of 1♥.
# TYPE minecraft_damage_dealt_total counter
minecraft_damage_dealt_total{player="ediri",type="absorbed"} 0
minecraft_damage_dealt_total{player="ediri",type="hit"} 610
minecraft_damage_dealt_total{player="ediri",type="resisted"} 0
# HELP minecraft_damage_received_total The amount of damage the player has received from different types in tenths of 1♥.
# TYPE minecraft_damage_received_total counter
minecraft_damage_received_total{player="ediri",type="absorbed"} 0
minecraft_damage_received_total{player="ediri",type="blocked_by_shield"} 0
minecraft_damage_received_total{player="ediri",type="resisted"} 0
minecraft_damage_received_total{player="ediri",type="taken"} 2295
```
